codepropertygraph/src/main/scala/io/shiftleft/Implicits.scala

```diff
@@ -1,7 +1,11 @@
 package io.shiftleft
 
+import org.apache.logging.log4j.LogManager
+
 object Implicits {
 
+  private val logger = LogManager.getLogger(classOf[Implicits])
```

Comment:
Objects are not types but it is not necessary to introduce a companion class just  for the purpose of the logger
```suggestion
  private val logger = LogManager.getLogger(getClass)
```